src/content/en/updates/2016/10/avoid-not-secure-warn.md

```diff
@@ -24,17 +24,20 @@ this warning.
 Warnings will be enabled by default for everyone in Chrome 56, slated for
 release in January 2017.
 
-To test the upcoming user experience before then, install the latest [Google
-Chrome Canary](https://www.google.com/chrome/browser/canary.html) build. At the
-time of this writing, Chrome Canary is version 56. Version 56.0.2895.0 is the
-first version to include detection of sensitive fields. You can see an
-example of the browser’s warning behavior on 
-[this page](http://http-password.badssl.com/) if you open it in Chrome
-56.0.2895.0 or later, then view the console in DevTools.
-
-When the Not Secure state is shown, the DevTools console shows the message `In
-Chrome M56 (Jan 2017), this page will be marked as “not secure” in the URL bar.
-For more information, see [https://goo.gl/zmWq3m](https://security.googleblog.com/2016/09/moving-towards-more-secure-web.html).`
+To test the upcoming user experience before that time, install the latest 
+[Google Chrome Canary](https://www.google.com/chrome/browser/canary.html) build.
```

Comment:
You need to mention the specific version of Chrome that is in canary so that it will make sense a year from now.